prost_build/src/main.rs

```diff
@@ -84,6 +84,8 @@ fn compile_protos(out_dir: &Path, tmp_dir: &Path) {
     let mut config = prost_build::Config::default();
     config.out_dir(tmp_dir);
     config
+        .type_attribute(".", "#[derive(serde::Deserialize, serde::Serialize)]")
+        .type_attribute(".", "#[serde(rename_all = \"snake_case\")]")
```

Comment:
Is there a reason to do this renaming? It seems like the best option for compatibility would be maintaining the default naming scheme.

Reply:
snake case has been the default in steward protos in the past, so I was trying to make it consistent.